src/applications/auth/containers/AuthApp.jsx

```diff
@@ -177,7 +178,7 @@ export class AuthApp extends React.Component {
     const view = this.state.error ? (
       this.renderError()
     ) : (
-      <LoadingIndicator message="Signing in to Vets.gov..." />
+      <LoadingIndicator message="Signing in to {siteName}..." />
```

Comment:
I think you need to make this a template string.

Reply:
Ah thank you!